docs/UserGuide.md

```diff
@@ -61,7 +83,48 @@ Format: `delete ingredient <index>`
 
 Example:
 1. `delete ingredient 1`
-    </br>Expected Output: `Your ingredient '500 units of tomato' has been deleted from the inventory.`
+    <br />Expected Output:<br />`Deleted Ingredient: flour; Quantity: 500.000 g; Expiry Date: 27-09-2021; Description: fresh; Tags: [expired]`
+
+### Edit Ingredients:
+`edit ingredient`
+
+Edit an ingredient from the inventory of ingredients
+
+Format: `edit ingredient <index> -<field flag> <new data>`
+
+Example: 
+1. `edit ingredient 1 -d juicy`
+<br />Expected Output:<br />`Edited Ingredient: Watermelon; Quantity: 1; Expiry Date: 20-10-2021; Description: juicy; Tags: [expiring][melon][water]`
+
+### Clear Ingredients:
+`clear`
+
+Clear all the ingredients from the inventory of ingredients
+
+Format: `clear`
```

Comment:
I think it should be 'clear ingredient' not clear, might have to update this

Reply:
This is based on our current implementation.